docs/src/build_config/cross_compilation.md

```diff
@@ -0,0 +1,31 @@
+# Cross Compilation
```

Comment:
Is this new section is going to be under the Build Configurations chapter?
Like this:
-- Build Configurations
-------Cross Compilation
If yes,  you need to update the SUMMARY.file as well. Some general instructions are here: https://github.com/tiiuae/ghaf/blob/main/docs/README.md#working-with-files.